src/WebJobs.Script/Description/DotNet/DependencyHelper.cs

```diff
@@ -165,5 +166,33 @@ public static List<string> GetRuntimeFallbacks(string rid)
             rids.AddRange(fallbacks.Fallbacks);
             return rids;
         }
+
+        /// <summary>
+        /// Checks if the string is in shared assembly representation format.
+        /// </summary>
+        /// <param name="assemblyFormatString"> string representing assembly information</param>
+        /// <returns> bool if string in shared assembly representation format. </returns>
+        public static bool IsSharedAssemblyFormat(string assemblyFormatString)
+        {
+           return assemblyFormatString != null && assemblyFormatString.StartsWith(AssemblyNamePrefix);
+        }
+
+        /// <summary>
+        /// Gets the Assembly name from the assembly path string, if in the expected format for a shared assembly.
+        /// </summary>
+        /// <param name="assemblyFormatString"> The assembly name string in the expected format. </param>
+        /// <returns> bool if the string was in the shared assembly format. </returns>
+        public static bool TryGetSharedAssembly(string assemblyFormatString, out string sharedAssembly)
```

Comment:
The concept of shared assemblies already exists, and this overloads that. We should use a different name here. Assembly reference might fit better.